### PR TITLE
Add contact form and footer of the portfolio mobile version

### PR DESCRIPTION
**Pull request by SufianShihab**
# PR Description
- Added contact form with full name, email, and message text field
- Added footer with social media icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,5 +246,61 @@ <h3 class="about-me__card__title">Skills</h3>
...
+          <form action="https://formspree.io/f/xknaqawg" class="contact__form">
+            <ul>
+              <li class="contact__form__element full-name">
+                <input type="text" id="full-name" name="full-name" placeholder="Full name" maxlength="30" required>
+              </li>
+              <li class="contact__form__element email">
+                <input type="email" id="email" name="email" placeholder="Email address" required>
+              </li>
+              <li class="contact__form__element message">
+                <textarea name="message" id="message" placeholder="Hey I’d like to connect to..."></textarea>
```
Comment (Shedrack-Sunday) [inline index.html:263]: - [ ] Kindly add a validation for the `textarea`  of `maxlength="500"` as specified on the project requirements. This is essential to keep your form at a reasonable point of input

File: index.html [hunk 2]
```diff
@@ -246,5 +246,61 @@ <h3 class="about-me__card__title">Skills</h3>
       </div>
     </div>
   </section>
+  <section id="contact">
+    <div class="contact">
+      <div class="contact__container">
+        <h2>I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.</h2>
+          <form action="https://formspree.io/f/xknaqawg" class="contact__form">
+            <ul>
```
Comment (Shedrack-Sunday) [inline index.html:255]: - [ ] Your form is not subitting because you're missing the `POST` method. Kindly add it up to "post" to the formspree server 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @SufianShihab     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudos on the validation so far ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by usorfaitheloho — APPROVED** [review]
Hi @SufianShihab  , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**